docs/dashboards/data-quality-dashboard.asciidoc

```diff
@@ -29,14 +29,18 @@ To use the Data Quality dashboard, you need at least the following {ref}/securit
 [discrete]
 [[data-quality-dash-check-indices]]
 == Check indices
-When you open the dashboard, data does not appear until you select indices to check.
+When you open the dashboard for the first time in a project, data does not appear until you (or another user) selects indices to check. 
```

Comment:
This is almost a nit that I let go, but it feels confusing. If I'm the user opening the dashboard, where is "another user" going to come from and select an index before I do? This edit focuses on what the reader alone is doing:
```suggestion
When you open the dashboard for the first time in a project, data does not appear until you select indices to check. 
```

Or, is the behavior actually based on _any user_ selecting an index _at any point_, in which case we don't need to mention opening the dashboard?
```suggestion
Data does not appear in the dashboard until a user selects indices to check. 
```

Reply:
this is great, thanks Joe